modules/operator-upgrade.adoc

```diff
@@ -34,59 +34,14 @@ In some cases, {productname} supports direct, single-step upgrades from prior (N
 . 3.4.z -> 3.7.z
 . 3.5.z -> 3.7.z
 
-For users on standalone deployments of Quay wanting to upgrade to 3.6, see the link:https://access.redhat.com/documentation/en-us/red_hat_quay/{producty}/html-single/upgrade_red_hat_quay/index#standalone_upgrade[Standalone upgrade] guide.
+For users on standalone deployments of Quay wanting to upgrade to 3.7, see the link:https://access.redhat.com/documentation/en-us/red_hat_quay/{producty}/html-single/upgrade_red_hat_quay/index#standalone_upgrade[Standalone upgrade] guide.
 
 
 === Upgrading Quay
 To update Quay from one minor version to the next, for example, 3.4 -> 3.5, you need to change the update channel for the Quay Operator.
 
 For `z` stream upgrades, for example, 3.4.2 -> 3.4.3, updates are released in the major-minor channel that the user initially selected during install. The procedure to perform a `z` stream upgrade depends on the `approvalStrategy` as outlined above. If the approval strategy is set to `Automatic`, the Quay Operator will upgrade automatically to the newest `z` stream. This results in automatic, rolling Quay updates to newer `z` streams with little to no downtime. Otherwise, the update must be manually approved before installation can begin.
 
-[[upgrade-33-36]]
-=== Notes on upgrading directly from 3.3.z or 3.4.z to 3.6
```

Comment:
I wonder if we need to leave this in - to upgrade from 3.3->3.7, you need to do 3.3->3.6 first, so either leave it in , or link out to this section in the 3.6 docs